node/src/GlideClusterClient.ts

```diff
@@ -811,12 +861,15 @@ export class GlideClusterClient extends BaseClient {
     public async fcallWithRoute(
         func: GlideString,
         args: GlideString[],
-        options?: RouteOption & DecoderOption,
+        options: RouteOption & DecoderOption,
```

Comment:
why?

Reply:
I tried to make route mandatory, but it is not required though.